save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityView.kt

```diff
@@ -172,3 +177,8 @@ private fun ChildrenBuilder.languageSpan(language: VulnerabilityLanguage) {
         +language.value
     }
 }
+
+internal fun hasRightsToEdit(currentUserInfo: UserInfo?, vulnerability: VulnerabilityDto) =
+    // globalRole is enriched on the App.kt level with "users/global-role" (based on ROLE from spring security authority)
+    // and is not stored in DB, so we cannot check userInfo directly, because in the getAllParticipants() users will have this field null
+    currentUserInfo.isSuperAdmin() || currentUserInfo?.originalLogins in vulnerability.getAllParticipants().map { it.originalLogins }
```

Comment:
why we check originalLogins? we can check only `currentUserInfo.name`

Reply:
yes, true, I misguided myself by thinking that it can be changed and will be reflected on the frontend but not on the backend. But it is not possible, you are right

Reply:
fixed